scripts/check_frontend_coverage.py

```diff
@@ -46,6 +46,7 @@
     'app.constants.ajs.ts',
     'app.constants.ts',
     'apply-validation.directive.ts',
+    'assets-backend-api.service.ts',
```

Comment:
Hmmm, shouldn't there be 3 additions to this file?

Reply:
`image-preloader.service.ts` and `url.service.ts` is already added in the whitelist.